Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
@ilovezfs However using `make install` in the formula would be less optimal than the current approve, because it would cause a additional `npm pack`.
Our current approve of installing the originally downloaded registry tarball of npm is faster.
Note this approve is only possible because we have the tarball still laying around and we know it's path, which is a assumption npm can't make in the more general case for `make install`.

BTW: Updating the formula to npm@5.2.0 would also require some little adjustments to the formula to expose the newly added `npx` to the homebrew users.